gandalf.js

```diff
@@ -2,127 +2,129 @@
  * Extras Plugin
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?